config.go

```diff
@@ -54,17 +54,17 @@ const (
 	defaultAllowHighFees       = false
 
 	// ticket buyer options
-	defaultMaxFee            dcrutil.Amount = 1e8
+	defaultMaxFee            dcrutil.Amount = 1e7
 	defaultMinFee            dcrutil.Amount = 1e6
 	defaultMaxPriceScale                    = 2.0
 	defaultMinPriceScale                    = 0.7
 	defaultAvgVWAPPriceDelta                = 2880
 	defaultMaxPerBlock                      = 5
 	defaultBlocksToAvg                      = 11
-	defaultFeeTargetScaling                 = 1.05
+	defaultFeeTargetScaling                 = 1.0
```

Comment:
One thing to consider is that we are increasing the fee paid even with just 1.0 scaling because it is moving up along with the median average. 1.01, is more competitive, as is 1.02, 1.03.